DSMAJCEexamples/src/main/java/DSMAJCE.java

```diff
@@ -0,0 +1,119 @@
+import com.fortanix.sdkms.jce.provider.SdkmsJCE;
+import com.fortanix.sdkms.jce.provider.keys.asym.rsa.RSAPrivateKeyImpl;
+import com.fortanix.sdkms.jce.provider.keys.asym.rsa.RSAPublicKeyImpl;
+import com.fortanix.sdkms.jce.provider.keys.sym.aes.SdkmsAESKey;
+import com.fortanix.sdkms.jce.provider.service.SdkmsKeyService;
+import com.fortanix.sdkms.v1.model.KeyObject;
+import com.fortanix.sdkms.v1.model.SobjectDescriptor;
+import static com.fortanix.sdkms.jce.provider.constants.ProviderConstants.*;
+
+import javax.crypto.*;
+import java.io.UnsupportedEncodingException;
+import java.nio.ByteBuffer;
+import java.security.*;
+import java.security.spec.MGF1ParameterSpec;
+import java.security.spec.PSSParameterSpec;
+import java.util.Arrays;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
```

Comment:
We should avoid using any external classes other than what is provided by Java itself.